server/src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryBuilder.java

```diff
@@ -312,10 +327,15 @@ public Query doToQuery(QueryShardContext context) {
                 throw new QueryShardException(context, "failed to find geo_point field [" + fieldName + "]");
             }
         }
-        if (!(fieldType instanceof GeoPointFieldType)) {
-            throw new QueryShardException(context, "field [" + fieldName + "] is not a geo_point field");
+//        if (!(fieldType instanceof GeoPointFieldType)) {
```

Comment:
Note that above refers just to geo_point ("failed to find geo_point field ["). That might need to be updated.